app/javascript/components/visualization/DotPlot.js

```diff
@@ -165,7 +156,7 @@ export function morpheusTabManager($target, dimensionsFn) {
     },
     setTabTitle: () => {},
     setActiveTab: () => {},
-    getWidth: () => {return dimensionsFn().width},
+    getWidth: () => $target.actual('width'),
```

Comment:
should we also use .actual for the line below out of caution?

Reply:
Yes, in fact you are correct.  Calling `actual(dimension)` will get the correct value regardless of whether the element is hidden or not.